libraries/core_libs/storage/src/storage.cpp

```diff
@@ -59,8 +64,10 @@ DbStorage::DbStorage(fs::path const& path, uint32_t db_snapshot_each_n_pbft_bloc
   if (db_revert_to_period) {
     recoverToPeriod(db_revert_to_period);
   }
-
-  checkStatus(rocksdb::DB::Open(options, db_path_.string(), descriptors, &handles_, &db_));
+  rocksdb::DB* db = nullptr;
```

Comment:
I think db should point to heap. Does it should delete db later?

Reply:
I am not deleting anything I am just creating empty pointer to db